rts/motoko-rts/src/types.rs

```diff
@@ -447,14 +455,15 @@ pub const TAG_CONCAT: Tag = 25;
 pub const TAG_NULL: Tag = 27;
 pub const TAG_ONE_WORD_FILLER: Tag = 29;
 pub const TAG_FREE_SPACE: Tag = 31;
+pub const TAG_REGION: Tag = 33;
```

Comment:
Just an idea (maybe not worth it): We could restructure the numbering such that `TAG_REGION` is less than the artificial values of `TAG_ONE_WORD_FILLER` and `TAG_FREE_SPACE` which are both no real objects.
E.g.
```
pub const TAG_REGION: Tag = 27;
pub const TAG_NULL: Tag = 29;
pub const TAG_ONE_WORD_FILLER: Tag = 31;
pub const TAG_FREE_SPACE: Tag = 33;
```

and in the compiler backend:

```
    | Region -> 27l
    | Null -> 29l
    | OneWordFiller -> 31l
    | FreeSpace -> 33l
    | ArraySliceMinimum -> 34l
```